assets/stylesheets/common/common.scss

```diff
@@ -1536,3 +1536,79 @@ $float-height: 530px;
     margin-left: 0.5em;
   }
 }
+
+.d-wrap[data-wrap="blocks"] {
```

Comment:
This css should probably be scoped inside chat message, in case core implements a similar `d-wrap`, right?

Reply:
qin any case that would cause issues but Im thinking of something more scoped for the named: bk-xxx